jenkins/api/Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-def PIPELINE_NAME = "issuer-kit-api"
+def PIPELINE_NAME = "api"
```

Comment:
Is this project deployed into a shared environment?  If so does the renaming of the Jenkins pipelines (like this one, there are others) cause any collisions with any other pipelines?